client/question/services/code_evaluators/PythonCodeRunnerService.js

```diff
@@ -106,20 +106,74 @@ tie.factory('PythonCodeRunnerService', [
       });
     };
 
+    var _sendCodeToServer = function(code, urlFragment) {
+      clearOutput();
+      return $http({
```

Comment:
You can use $http.post(url, params). That's what all the common tie config stuff was for.